### ISSUE TITLE
Is it possible to make use of proxy methods?

### ISSUE THREAD

**Issue by SimonVillage** [centrifugal/centrifuge-dart#55]
Hi,

I am trying to implement `proxy_subscribe_endpoint`. The endpoint gets called on my backend on subscribe, I return a message like this:

```
{
  "result": {
    "data": {
      "hello": "world"
    }
  }
}
```

However, on the `subscribeSuccessStream` event I only receive `SubscribeSuccessEvent{isResubscribed: false, isRecovered: false}`

Is this functionality implemented already? I am trying to respond to the subscribing user with some initial user information.

**Comment by FZambia** [comment]
Sorry, the bug in a server – fixed by https://github.com/centrifugal/centrifugo/commit/fc07520fabe25026a98e85f00b69a4a4ab6ac7ae – will be available in next Centrifugo release.

**Comment by FZambia** [comment]
Released both Centrifugo v3.0.3 and centrifuge-dart v0.7.1 – returning `data` from subscribe proxy must be proxied towards `data` field of `SubscribeSuccessEvent`.
